client/blocks/jetpack-connect-skip-user/index.js

```diff
@@ -32,23 +33,29 @@ class JetpackConnectSkipUser extends Component {
 	render() {
 		return (
 			<div className="jetpack-connect-skip-user__userless-form">
-				<h2>Or start using Jetpack now</h2>
+				<h2>{ this.props.translate( 'Or start using Jetpack now' ) }</h2>
 
 				<p>Jump in and start using Jetpack right away.</p>
 				<p>
-					<a target="_blank" href="https://jetpack.com/support/features/" rel="noreferrer">
-						Some features
+					<a
+						target="_blank"
+						href="https://jetpack.com/support/why-the-wordpress-com-connection-is-important-for-jetpack/"
+						rel="noreferrer"
+					>
+						{ this.props.translate( 'Some features' ) }
```

Comment:
This is a little more difficult to translate, because even though this string is fine (`Some features`), the next string (`will not be available, but...`) does not stand alone. Translation occurs per-string, so it's important for each string to have the full context. Really, this is all one sentence (`Some features will not be available, but...`), but it contains a link on some of the text.

One way we can do that is to use component interpolation, which is a feature that `translate()` supports. So this could become something like (I always have to look this syntax up):

```js
this.props.translate( '{{link}}Some features{{/link}} will not be available, but...',
  { components: { link: <a target="_blank" href="..." rel="noreferrer" > } }
)
```

Reply:
Took a try at this in `0c1aee2` - I think that output is expected/better/correct?

```
/* THIS IS A GENERATED FILE. DO NOT EDIT DIRECTLY. */
$undefined_i18n_strings = array(
__( "Or start using Jetpack now" ), // client/blocks/jetpack-connect-skip-user/index.js:38
__( "Jump in and start using Jetpack right away." ), // client/blocks/jetpack-connect-skip-user/index.js:40
__( "{{link}}Some features{{/link}} will not be available, but you'll be able to connect your user account at any point to unlock them." ), // client/blocks/jetpack-connect-skip-user/index.js:42
__( "Continue without user account" ), // client/blocks/jetpack-connect-skip-user/index.js:59
);
```

Reply:
That looks right to me!